docs/quickstarts/add-machine-pool /metadata.yml

```diff
@@ -1,5 +1,5 @@
 kind: QuickStarts
-name: rosa-add-machine-pool
+name: rosa-osd-add-machine-pool
```

Comment:
The content filename does not match this value.
```
13:49:17 panic: open docs/quickstarts/add-machine-pool /rosa-osd-add-machine-pool.yaml: no such file or directory

```
The validation fails because its trying to open a file called: rosa-osd-add-machine-pool.yaml